app/views/tags/feed.atom.builder

```diff
@@ -6,10 +6,10 @@ atom_feed do |feed|
     unless work.unrevealed?
       feed.entry work do |entry|
         entry.title work.title 
```

Comment:
Why are you caching summary but not title? would it maybe make more sense to cache/expire the entire entry together? When does expire_caches get called -- will it definitely change whenever the authors are changed? Would like to see a test that checks the feed before/after an author change, title change, summary change.